src/qt/transactionrecord.h

```diff
@@ -61,8 +61,8 @@ class TransactionStatus
                       finalization */
     /**@}*/
 
-    /** Current number of blocks (to know whether cached status is still valid) */
-    int cur_num_blocks;
+    /** Current block hash (to know whether cached status is still valid) */
+    uint256 cur_block_hash;
```

Comment:
@furszy may I suggest the default member initialization again
```suggestion
    /** Current block hash (to know whether cached status is still valid) */
    uint256 cur_block_hash{};
```
?
And drop `cur_block_hash(uint256())` in the ctor initialization list.

Reply:
And use `m_` prefix for the data member `cur_block_hash` name?

Reply:
yeah done.